.github/workflows/gradle.yml

```diff
@@ -855,158 +863,207 @@ jobs:
-  #
-  #      - name: Run Helm End-to-End Acceptance Tests
-  #        env:
-  #          USER: root
-  #          HOME: /home/runner
-  #          # AWS_S3_INTEGRATION_TEST_CREDS can be found in LastPass as AWS_S3_INTEGRATION_TEST_CREDS
-  #          AWS_S3_INTEGRATION_TEST_CREDS: ${{ secrets.AWS_S3_INTEGRATION_TEST_CREDS }}
-  #          SECRET_STORE_GCP_CREDENTIALS: ${{ secrets.SECRET_STORE_GCP_CREDENTIALS }}
-  #          SECRET_STORE_GCP_PROJECT_ID: ${{ secrets.SECRET_STORE_GCP_PROJECT_ID }}
-  #        run: |
-  #          CI=true IS_MINIKUBE=true ./tools/bin/acceptance_test_kube_helm.sh
-  #
-  #      - uses: actions/upload-artifact@v3
-  #        if: failure()
-  #        with:
-  #          name: Kubernetes Logs
-  #          path: /tmp/kubernetes_logs/*
+  helm-acceptance-test:
+    name: "Platform: Acceptance Tests (Helm)"
+    # In case of self-hosted EC2 errors, removed the `needs` line and switch back to running on ubuntu-latest.
+    needs: [start-helm-acceptance-test-runner] # required to start the main job when the runner is ready
+    runs-on: ${{ needs.start-helm-acceptance-test-runner.outputs.label }} # run the job on the newly created runner
+    # this is the label of the runner
+    environment: more-secrets
+    timeout-minutes: 90
+    defaults:
+      run:
+        working-directory: "/actions-runner/_work/airbyte/airbyte"
+    env:
+      HOME: /actions-runner/_work/airbyte/airbyte
+      GITHUB_WORKSPACE: /actions-runner/_work/airbyte/airbyte
+    # github:
+    #   workspace: /home/runner
+
+    steps:
+      - name: Fix EC-2 Runner
+        run: |
+          mkdir -p /actions-runner/_work/airbyte/airbyte && mkdir -p /actions-runner/_work/airbyte/airbyte/.kube
+        
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Delete default old docker and replace it with a new one
+        shell: bash
+        run: |
+          sudo apt-get remove docker.io || sudo apt-get remove docker
+          curl -fsSL https://get.docker.com | bash -
+
+      - name: Cache Build Artifacts
+        uses: ./.github/actions/cache-build-artifacts
+        with:
+          cache-key: ${{ secrets.CACHE_VERSION }}
+          cache-python: "false"
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9' 
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "17"
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "lts/gallium"
+
+      - name: Install unzip
+        shell: bash
+        run: sudo apt-get update && sudo apt-get install -y unzip
+
+      - uses: azure/setup-helm@v3
+        with:
+          version: 'latest'
+          token: ${{ secrets.GITHUB_TOKEN }}
+        id: install
+
+      - uses: azure/setup-kubectl@v3
+        with:
+          version: 'v1.21.2' # default is latest stable
+
+      - name: Install tmate and it's dependencies
```

Comment:
and this is fine and dandy, but I think there is a tmate action that does more or less all of this

Reply:
Tmate actions take runtime space during workflow execution, while this method of running tmate just spins the daemon and we can access the instance during test execution. As for flaws, there's only one, when workflow dies, connection does too.